auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -429,73 +517,30 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     return this.selectedColumns.includes(col)
   }
 
-  /** Custom sorting method to handle consolidated Nr and Affiliations data. */
-  private customSort (items, index, isDesc): any {
-    items.sort((a, b) => {
-      switch (index[0]) {
-        case 'lastModified':
-          let dateA, dateB
-          if (a.lastModified) {
-            dateA = a.lastModified
-          } else {
-            dateA = a.modified
-          }
-          if (b.lastModified) {
-            dateB = b.lastModified
-          } else {
-            dateB = b.modified
-          }
+  private filter (items: Business[]): Business[] {
+    let newItems = [...items]
+    for (const i in this.getHeaders) {
+      const header = this.getHeaders[i]
+      if (this.filterParams[header.value]) this.filterActive[header.value] = true
+      if (header.filterFn && this.filterActive[header.value]) {
+        newItems = newItems.filter((item) => {
+          if (header.filterType === 'select') return this.selectFilter(header.filterFn(item), this.filterParams[header.value])
+          else return this.textFilter(header.filterFn(item), this.filterParams[header.value])
+        })
+      }
+      if (!this.filterParams[header.value]) this.filterActive[header.value] = false
+    }
+    return newItems
+  }
 
-          if (!isDesc[0]) {
-            return +new Date(dateB) - +new Date(dateA)
-          } else {
-            return +new Date(dateA) - +new Date(dateB)
-          }
-        case 'name':
-          let nameA, nameB
-          if (a.nameRequest) {
-            nameA = a.nameRequest?.names[0].name
-          } else {
-            nameA = this.isNumberedIncorporationApplication(a) ? 'Numbered Benefit Company' : a.name
-          }
-          if (b.nameRequest) {
-            nameB = b.nameRequest?.names[0].name
-          } else {
-            nameB = this.isNumberedIncorporationApplication(b) ? 'Numbered Benefit Company' : b.name
-          }
+  private selectFilter = (itemVal: string, filterVal: string) => {
+    if (filterVal) return itemVal.toUpperCase() === filterVal.toUpperCase()
```

Comment:
return filterVal &&  itemVal.toUpperCase() === filterVal.toUpperCase()

Reply:
in this case I actually want it to be `return !filterVal || itemVal.toUpperCase() === filterVal.toUpperCase()`

Reply:
yup, forgot about the return true